presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -234,7 +237,7 @@ public ThriftMetastoreStats getStats()
     }
 
     @Override
-    public Optional<Table> getTable(String databaseName, String tableName)
+    public Optional<Table> getTable(HiveIdentity identity, String databaseName, String tableName)
```

Comment:
I'll update this method to pass this identity to getTableFromMetastore and alternativeCall.

Reply:
Which identity do you mean?